spree_bank_transfer.gemspec

```diff
@@ -16,9 +16,9 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_dependency 'spree_core', '~> 3.1.0'
+  s.add_dependency 'spree_core', '~> 3.2.0.rc1'
```

Comment:
@pikender This can be moved to 3.2.0.rc2 as it's already released and much more stable than rc1